backend/src/main/java/net/finman/dao/InvoiceDao.java

```diff
@@ -40,12 +41,12 @@
     List<Invoice> getInvoices(String source) throws ResourceNotFoundException;
 
     /**
-     * 
+     *
      * @param invoice An invoice
      * @param seller  Email of the organisation that sent the invoice
      * @return A list of items in an invoice
      * @throws ResourceNotFoundException if there wasn't any items found in an
      *                                   invoice or if the get-method fails.
      */
-    List<Item> getInvoiceItems(UUID invoice, String seller) throws ResourceNotFoundException;
+    List<InvoiceItem> getInvoiceItems(UUID invoice, String seller) throws ResourceNotFoundException;
```

Comment:
We don't need this function getInvoiceItems(...) anymore since invoiceItems are retrieved through an Invoice directly